src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -683,4 +708,136 @@ public function column_cb( $post ) {
 			<div class="locked-indicator"></div>
 		<?php
 	}
+
+	/**
+	 * Displays the pagination.
+	 *
+	 * @since TBD
+	 * @access protected
+	 *
+	 * @param string $which Equal to NULL, 'top' or 'bottom'.
+	 */
+	protected function pagination( $which ) {
+		if ( empty( $this->_pagination_args ) ) {
+			return;
+		}
+
+		$total_items     = $this->_pagination_args['total_items'];
+		$total_pages     = $this->_pagination_args['total_pages'];
+		$infinite_scroll = false;
+		if ( isset( $this->_pagination_args['infinite_scroll'] ) ) {
+			$infinite_scroll = $this->_pagination_args['infinite_scroll'];
+		}
+
+		if ( 'top' === $which && $total_pages > 1 ) {
+			$this->screen->render_screen_reader_content( 'heading_pagination' );
+		}
+
+		$output = '<span class="displaying-num">' . sprintf(
+			/* translators: %s: Number of items. */
+			_n( '%s item', '%s items', $total_items, 'the-events-calendar' ),
+			number_format_i18n( $total_items )
+		) . '</span>';
+
+		$current              = $this->get_pagenum();
+		$removable_query_args = wp_removable_query_args();
+
+		$current_url = set_url_scheme( 'http://' . esc_url_raw( $_SERVER['HTTP_HOST'] ) . esc_url_raw( $_SERVER['REQUEST_URI'] ) );
+
+		$current_url = remove_query_arg( $removable_query_args, $current_url );
+
+		if ( isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
```

Comment:
`isset` not required.  as `empty` does the same check. 

```suggestion
		if ( ! empty( $_REQUEST['s'] ) ) {
```